packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -87,26 +87,36 @@ def test_if_dcos_history_service_is_up(dcos_api_session):
 
 
 def test_if_marathon_ui_is_up(dcos_api_session):
-    r = dcos_api_session.get('/marathon/ui/')
+    r = dcos_api_session.get('/marathon/v2/info')
 
     assert r.status_code == 200
-    assert len(r.text) > 100
-    assert '<title>Marathon</title>' in r.text
+    response_json = r.json()
+    assert "name" in response_json
+    assert "marathon" == response_json["name"]
+    assert "version" in response_json
+    assert response_json["version"].startswith("1.6.0")
```

Comment:
Why are we hardcoding software version in the test?

Reply:
* I was trying to be more specific, with the marathon version that DCOS is shipping.
* If marathon upgrades the version with a bump, I wanted a test to track it, so that could come here and update the test.

Reply:
@vespian  - I am open removing this, I wanted to see what could be asserted in response.json and added it. If you are -1 on it, I am fine...

Reply:
Let's please remove it.

The test is checking if the root Marathon is up, tracking marathon version is already done via `package/*/buildinfo.json`.

Can we also change the test name please? It no longer checks if the marathon UI is available - only whether marathon itself is up. I also wonder if we should be checking for the availability of the Marathon UI. @kensipe - do you think this should be covered as well?

Reply:
SGTM. 

- I will rename the method to check marathon_is_up.

- BTW, tracking marathon version is not done in a `test` Similar to how [dcos version is tested](https://github.com/dcos/dcos/blob/master/dcos_installer/test_backend.py#L105).

- It will be good to know if marathon provides a UI endpoint. It's definitely not listed in their API documentation.

Reply:
Verified in product. `/marathon` redirects to `/marathon/ui` 

It does not return any `json` output.

Reply:
I agree with @vespian that we must not test for specific version in our tests. That's why packaging selects an specific commit. Neither do we want to add an extra step every time we do a code bump. So I will prefer us to remove that assertion.